launch

```diff
@@ -404,6 +404,11 @@ Please ensure to execute tizen command if use WGT packaging.''')
     % (pkg_manager, pkg_manager), shell=True)
   return os.path.dirname(tizen_path)
 
+def build_brackets():
+  subprocess.check_call('cd libs/brackets-server/brackets-dist 2>/dev/null' \
```

Comment:
Add try-catch to check build fail.
You can check code in 'run_service()'.
 In another way, How about moving the brackets build code in the 'run_service()' to front of the 'dry_run()'?

Reply:
done,